### ISSUE TITLE
Export failure

### ISSUE THREAD

**Issue by winer222** [space928/Blender-O3D-IO-Public#35]
### Describe the bug
Export failure

### To Reproduce
Steps to reproduce the behaviour:
1. Tried to export in 279
2. Failed

### Expected behaviour
Export successfully...

### Additional context
```
location: <unknown location>:-1
Traceback (most recent call last):
  File "Blender\2.79\scripts\addons\Blender-O3D-IO__init__.py", line 158, in execute
    ).to_4x4() @ Matrix.Scale(self.global_scale, 4)
TypeError: unsupported operand type(s) for @: 'Matrix' and 'Matrix'
```

**Comment by space928** [comment]
I really need to write a test suite...  
The `@` matrix multiply operator is not supported in the version of python used by 2.79 and must be replaced with `*`, but only for 2.79. Looking through the code, I think I missed a few of these (I don't test much on 2.79). In every case the fix should look something like:
```py
        if bpy.app.version[0] < 3 and bpy.app.version[1] < 80:
            mx = mx * Matrix.Scale(-1.0, 4, (0.0, 1.0, 0.0))
        else:
            mx = mx @ Matrix.Scale(-1.0, 4, (0.0, 1.0, 0.0))
```

**Comment by space928** [comment]
Fixed as of 03a979c1f665f5065dc7443ea06bba98285789eb
